my-app/src/API/index.ts

```diff
@@ -0,0 +1,8 @@
+export const API_KEY: string = 'bb2b538ec7dc87a9a9ecaa9065c3baff';
```

Comment:
It's not a good idea to put api keys in source code of project, find a possibility to do it in another way. Common approach with `webpack dotenv` for example.

Reply:
done